widgets/web-component-widgets/web-component/README.md

```diff
@@ -1,26 +1,38 @@
 # Web component
 
-This sample demonstrates how to write a Web component and include it in a custom Experience Builder widget.
+This sample shows how to create a Web Component and embed it in a custom ArcGIS Experience Builder widget.
 
-## How to use the sample
-Clone the [sample repo](https://github.com/esri/arcgis-experience-builder-sdk-resources) and copy this widget's folder (within `widgets`) to the `client/your-extensions/widgets` folder of your Experience Builder installation.
+## Contents
+
+- `manifest.json` — Widget metadata and Experience Builder version compatibility
+- `src/runtime/widget.tsx` — Functional runtime component; imports the Web Component, Calcite Components, and map components
+- `src/setting/setting.tsx` — Builder settings panel to select a target Map widget
+- `src/runtime/my-component.js` — Custom Web Component definition registered with window.customElements
+
+## Prerequisites
+
+- ArcGIS Experience Builder Developer Edition installed and running (version compatible with the widget’s manifest).
 
 ## How it works
 
-A [Web component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) is defined in `my-component.js`. This is a JavaScript file - not a TypeScript file - because Experience Builder is compiling the `.ts` files to ES5, but Web components work on >= ES6 only. The first line uses the [ECMAScript 2015 class syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) to create a new Class that extends from HTMLElement.
+The custom element is implemented in the JavaScript file `my-component.js`. It is written in plain JavaScript because Experience Builder transpiles `.ts` files to ES5, while Web Components require ES6 or newer. The file begins with an ECMAScript 2015 (ES6) class that extends `HTMLElement`, which defines the Web Component’s behavior.
```

Comment:
The compile target is ES6 now. @blissvisitor can you upgrade the widget to `.ts`?

Reply:
Thanks. Got it fixed.